csvqb/csvqb/models/cube/csvqb/components/attribute.py

```diff
@@ -92,3 +145,17 @@ def from_data(
 
     def validate_data(self, data: PandasDataTypes) -> List[ValidationError]:
         return []  # TODO: implement this
+
+
+@dataclass
+class NewQbAttributeLiteral(NewQbAttribute, QbAttributeLiteral):
+    new_attribute_values: List[NewQbAttributeValue] = field(
+        default_factory=list, repr=False
+    )
+
+    def validate_data(self, data: PandasDataTypes) -> List[ValidationError]:
+        errors = []
+
+        errors.append(*[NewQbAttribute.validate_data(self, data)])
```

Comment:
Again, does this need wrapping in another list? The function already returns a list.